src/djangosaml2_spid/identity_providers.py

```diff
@@ -0,0 +1,17 @@
+# source: https://registry.spid.gov.it/identity-providers
+
+import urllib.request
+import json
+
+identity_providers_url = 'https://registry.spid.gov.it/assets/data/idp.json'
+
+
+def download_identity_providers():
+    with urllib.request.urlopen(identity_providers_url) as response:
```

Comment:
https TLS/SSL validity must be checked here, better use python requests

Reply:
On it! 👍🏻

Reply:
I wrote a first version of the utility and added instruction to the README.md, check it out! 😄